Update memory dump command with download options (#630)
kubevirt/user-guide: docs/operations/memory_dump.md

A PVC with size big enough for the dump will be created. We can also request specific storage class and access mode with appropriate flags.

#### Download memory dump

> By adding `--output` flag the memory will be dumped to the pvc and then downloaded to the given output path.

Reviewer: ```suggestion
By adding the `--output` flag, the memory will be dumped to the PVC and then downloaded to the given output path.
```

```bash
$ virtctl memory-dump get myvm --claim-name=memoryvolume --create-claim --output=memoryDump.dump.gz
```

> For downloading the last memory dump from the PVC associated to the VM, without triggering another memory dump use the memory dump donwload command

Reviewer: ```suggestion
For downloading the last memory dump from the PVC associated with the VM, without triggering another memory dump, use the memory dump download command.
```

```bash
$ virtctl memory-dump download myvm --output=memoryDump.dump.gz
```

> For downloading memory dump from a PVC already removed from the VM association you can use the [virtctl vmexport command](https://github.com/kubevirt/user-guide/blob/main/docs/operations/export_api.md)

Reviewer: ```suggestion
For downloading a memory dump from a PVC already removed from the VM association you can use the [virtctl vmexport command](https://github.com/kubevirt/user-guide/blob/main/docs/operations/export_api.md)
```

Also, would it make sense to rephrase "removed from the VM association" as "disassociated from the VM" ?
